release-tools/src/main/java/org/springframework/data/release/build/MavenBuildSystem.java

```diff
@@ -400,12 +449,34 @@ private void deployToMavenCentral(ModuleIteration module) {
 
 		logger.log(module, "Deploying artifacts to Sonatype OSS Nexus…");
 
-		CommandLine arguments = CommandLine.of(Goal.CLEAN, Goal.DEPLOY, //
-				profile("ci,release,central"), //
-				SKIP_TESTS, //
-				arg("gpg.executable").withValue(gpg.getExecutable()), //
-				arg("gpg.keyname").withValue(gpg.getKeyname()), //
-				arg("gpg.password").withValue(gpg.getPassword()));
+		final CommandLine arguments;
+
+		mvn.getProperties().setConsoleLogger(true);
+
+		if (properties.getLocal()) {
```

Comment:
1. We need the logs for `rc-open` to get piped into a file, because parsing it is the only way to get ahold of the staging repository ID value generated.

When running tools locally, writing outputs into log files is fine. But when run on CI, it's preferable to instead write to console where we can read it (can't grab log files after the fact).

2. Between CI server and locally, the tools have different expectations of where to find the GPG keyfile and how to properly interact it with. I'm frankly not sure if we need to drop "local" from deployment and simply presume all releases are going through CI. If so, I could remove this toggle I've set up. I agree it does make things confusing.

Reply:
Looks like this was a debugging construct.